docs/development.md

```diff
@@ -3,17 +3,15 @@
 Install required tools:
```

Comment:
Let's also update this section with a summary of the steps: https://github.com/jmorganca/ollama#building

Something like:
* Install `cmake` or (optionally, required tools for GPUs)
* run `go generate ./...`
* run `go build .`
```